cmd/config/internal/commands/cmdlistsetters.go

```diff
@@ -123,8 +124,17 @@ func (r *ListSettersRunner) ListSetters(w io.Writer, openAPIPath, resourcePath s
 		} else {
 			required = "No"
 		}
+
+		var isSet = "No"
+		if s.IsSet != "" {
+			var b, _ = strconv.ParseBool(s.IsSet)
```

Comment:
We should not ignore error here. What if the string is not one of the accepted boolean values? We should bubble that up.

Reply:
you can use [short rep](https://tour.golang.org/basics/10) `b, err := strconv.ParseBool(s.IsSet)` instead of declaring as a var.

Reply:
Done.